addons/binding/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/action/LGWebOSActions.java~

```diff
@@ -0,0 +1,384 @@
+/**
```

Comment:
This file seems to be an accident...

Reply:
oh yes, accidentally committed those emacs temp files